src/plugins/videoroom-plugin.js

```diff
@@ -52,6 +53,7 @@ const PLUGIN_EVENT = {
   LEAVING: 'videoroom_leaving',
   UPDATED: 'videoroom_updated',
   KICKED: 'videoroom_kicked',
+  ENABLE_RECORDING: 'videoroom_enable_recording',
```

Comment:
We use the name of the events in the past form.
Also since the semantics are actually included in the boolean of the payload, a more accurate name for the event could be:

`RECORDING_ENABLED_STATE:  videoroom_recording_enabled_state`